MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -2413,6 +2415,7 @@
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = GJZR2MEM28;
```

Comment:
Might not be necessary?

Reply:
It's the same team as Example-Swift and Example-Objective-C are using, and it's needed to run on a device.